vms/platformvm/cache_versioned_state.go

```diff
@@ -81,7 +84,8 @@ type versionedStateImpl struct {
 
 	timestamp time.Time
 
-	currentSupply uint64
+	currentSupply       uint64
+	validatorBondAmount uint64
```

Comment:
What is the reason this one is copied from genesis to state? IIRC we always can access genesis params.
Is the intention to have it updateable in the future through TX ?

Reply:
> Is the intention to have it updateable in the future through TX ?
yes, as was stated in PR description